components/i2c/bme280/definition.json

```diff
@@ -0,0 +1,28 @@
+{
+    "displayName":"BME280",
+    "i2cAddresses":[
+       "0x76, 0x77"
+    ],
+    "subcomponents":[
+       {
+          "displayName":"Temperature Sensor",
+          "sensorType":"ambient-temp",
+          "defaultPeriod":900
+       },
+       {
+          "displayName":"Humidity Sensor",
+          "sensorType":"humidity",
+          "defaultPeriod":900
+       },
+       {
+          "displayName":"Pressure Sensor",
+          "sensorType":"pressure",
+          "defaultPeriod":900
+       },
+       {
+          "displayName":"Altitude Sensor",
```

Comment:
if we add this we have to note that we assume the sea level pressure so its relative changes not absolute

Reply:
@ladyada In the future, we'll have an "advanced configuration" on the form for the user to add their location's sea level pressure... For now, are you OK with changing this `displayName` to "Altitude Sensor (Relative)", or omitting it from the form entirely?

Reply:
yep thats a good solution!

Reply:
great, resolved in https://github.com/adafruit/Wippersnapper_Components/pull/12/commits/3494f6e1911d3bbc7096cffe3d43c546e9c34890